src/main/java/eggventory/items/Stock.java

```diff
@@ -16,7 +17,7 @@
     private int lost;
```

Comment:
We should put a size limit on the quantity and description so that it does not exceed the box of the ui